pkgs/development/python-modules/django-leaflet/default.nix

```
@@ -0,0 +1,32 @@
+{ pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  pname = "django-leaflet";
+  version = "0.29.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-L23Ixxh/0i5itvK3tC7thpIPgb7DEqplSYHr5byOCGY=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    django
+  ];
+
+  # The tests seem to be impure.
+  # They are throwing a error about unset configs:
+  # > django.core.exceptions.ImproperlyConfigured: Requested setting LEAFLET_CONFIG, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
+  doCheck = false;
+
+  meta = with pkgs.lib; {
+    description = "allows you to use Leaflet in your Django projects";
+    homepage = "https://github.com/makinacorpus/django-leaflet";
+    license = licenses.gpl3Only;
```

License seem to be [`LGPL3+`](https://github.com/makinacorpus/django-leaflet/blob/0.29.0/COPYRIGHT#L3-L6).

```suggestion
    license = licenses.lgpl3Plus;
```